dev/benchmarking/ddl/2_00_views.sql

```diff
@@ -0,0 +1,236 @@
+/*
+  Licensed to the Apache Software Foundation (ASF) under one
+  or more contributor license agreements.  See the NOTICE file
+  distributed with this work for additional information
+  regarding copyright ownership.  The ASF licenses this file
+  to you under the Apache License, Version 2.0 (the
+  "License"); you may not use this file except in compliance
+  with the License.  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing,
+  software distributed under the License is distributed on an
+  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  KIND, either express or implied.  See the License for the
+  specific language governing permissions and limitations
+  under the License.
+*/
+
+-- MACHINE_VIEW
+CREATE OR REPLACE VIEW public.machine_view AS
+  SELECT
+    machine.machine_id
+    , machine_name
+    , memory_bytes
+    , cpu_actual_frequency_Hz
+    , os_name
+    , architecture_name
+    , kernel_name
+    , cpu_model_name
+    , cpu_core_count
+    , cpu_thread_count
+    , cpu_frequency_max_Hz
+    , cpu_frequency_min_Hz
+    , cpu_L1d_cache_bytes
+    , cpu_L1i_cache_bytes
+    , cpu_L2_cache_bytes
+    , cpu_L3_cache_bytes
+    , cpu_RAM_bytes
```

Comment:
Does not make much sense to have this in addition to `memory_bytes`.

Reply:
Excellent. It is gone.